src/common/utils/utility.js

```diff
@@ -64,3 +62,9 @@ export const moveToDeriv = async () => {
         window.location.replace('https://binary.com/move-to-deriv');
     }
 };
+
+export const getClientsCountryByIP = async () => {
```

Comment:
We have saved the country on localStorage. I think you can remove these codes and use the storage.

Reply:
unfortunately country in localStroage is only available for logged clients, and i need to check it for non-logged clients as well